docs/getting-started/overview.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Substrate + AssemblyScript = Subsembly
+---
+
+# Overview
+
+**`Subsembly`** framework used for developing [**`Substrate`**](substrate-essentials.md) runtimes in **`AssemblyScript`**. The project is work in progress.
+
+This is the high-level overview of **`Subsembly`**.
```

Comment:
This is the high-level overview of **`Subsembly`**. -> `The following diagram shows the high-level components Architecture of **`Subsembly`**.`